test/sql/hnsw_index_from_file.sql

```diff
@@ -0,0 +1,81 @@
+------------------------------------------------------------------------------
+-- Test HNSW index creation from file
+------------------------------------------------------------------------------
+
+-- Index files were created with ldb-create-index program which source is under https://github.com/lanterndata/lanterndb_extras/
+-- We have exported index files for sift1k dataset for cosine and l2sq distances
+-- With the following params m=16 ef=64 ef_construction=128 dims=128
+-- Validate that index creation works with a small number of vectors
+\ir utils/sift1k_array.sql
+
+-- Validate error on invalid path
+\set ON_ERROR_STOP off
+CREATE INDEX hnsw_l2_index ON sift_base1k USING hnsw (v dist_l2sq_ops) WITH (dims=128, M=16, ef=64, ef_construction=128, _experimental_index_path='/tmp/lanterndb/files/invalid-path');
+\set ON_ERROR_STOP on
+-- Validate that creating an index from file works
+CREATE INDEX hnsw_l2_index ON sift_base1k USING hnsw (v dist_l2sq_ops) WITH (dims=128, M=16, ef=64, ef_construction=128, _experimental_index_path='/tmp/lanterndb/files/index-sift1k-l2.usearch');
+SELECT * FROM ldb_get_indexes('sift_base1k');
+
+SET enable_seqscan = false;
+
+SELECT v AS v777 FROM sift_base1k WHERE id = 777 \gset
+EXPLAIN (COSTS FALSE) SELECT ROUND(l2sq_dist(v, :'v777')::numeric, 2) FROM sift_base1k order by v <-> :'v777' LIMIT 10;
+SELECT ROUND(l2sq_dist(v, :'v777')::numeric, 2) FROM sift_base1k order by v <-> :'v777' LIMIT 10;
+-- Validate that inserting rows on index created from file works as expected
+INSERT INTO sift_base1k (v) VALUES 
+('{1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1}'), 
```

Comment:
this can be `INSERT INTO sift_base1k (id, v) VALUES (1001, '{...}') so it is more clear that the id of the vector is 1001.

And, instead of typing out the vector explicitly, you can probably use another vector from the existing table.
E.g. SELECT v AS v777 FROM sift_base1k WHERE id = 777 \gset
then insert the value :'v777'